JSON/Forget Any Color.json

```diff
@@ -58,5 +58,6 @@
   "Souvenir": { "Status": "Supported" },
   "SteamID": "2126185549",
   "Symbol": "Fac",
+  "TutorialVideoUrl": "https://www.youtube.com/watch?v=g6qQcHsyeBA&t",
```

Comment:
Remove `&t`

Reply:
In the following days a major update will be released, changing the rules and leaving parts of this tutorial outdated. 

For now, this should be excluded.